scripts/variantstore/wdl/GvsPrepareCallset.wdl

```diff
@@ -72,6 +76,7 @@ task PrepareCallsetTask {
             --destination_table ~{destination_cohort_table_name} \
             --fq_cohort_sample_names ~{fq_cohort_sample_table} \
             --query_project ~{query_project} \
+            ~{sep=" " query_label_args} \
```

Comment:
shouldn't the separator here be "--query_labels"?

Reply:
I think --query-labels is embedded in query_label_args (since it's in the prefix)